Kernel/System/Fred/STDERRLog.pm

```diff
@@ -113,13 +113,6 @@ sub DataGet {
         shift @LogMessages;
     }
 
```

Comment:
But this isn't a duplicated block. Notice the **reverse**. Reversing the line is also needed to counteract the reverse from line 101.

Reply:
But now I'm getting confused. The loop is looking at the and of the array @LogMessages but removing at the front. And modifying an array while looping over it, is a bad idea anyways. I would do something simple like:
`my @TrimmedLogMessage = grep { $_ !~ m/ \A \s+ \z }xms } @LogMessages; `
Assuming that there are no empty lines in between.